src/ert/gui/model/snapshot.py

```diff
@@ -211,9 +212,27 @@ def _add_partial_snapshot(self, partial: PartialSnapshot, iter_: int):
                 if job.index:
                     job_node.data[ids.INDEX] = job.index
                 if job.current_memory_usage:
-                    job_node.data[ids.CURRENT_MEMORY_USAGE] = job.current_memory_usage
+                    job_node.data[ids.CURRENT_MEMORY_USAGE] = float(
+                        job.current_memory_usage
+                    )
+                    real_node.data[ids.CURRENT_MEMORY_USAGE] = max(
+                        real_node.data.get(ids.CURRENT_MEMORY_USAGE, -1),
+                        float(job.current_memory_usage),
+                    )
+                    self.root.data[ids.CURRENT_MEMORY_USAGE] = max(
+                        self.root.data.get(ids.CURRENT_MEMORY_USAGE, -1),
+                        float(job.current_memory_usage),
+                    )
```

Comment:
I assume the two first are for the different views, but what is the last entry?

Reply:
Will the current memory usage ever go down with this? If you always select max, won't the current always be the same as maximum?

Reply:
Yes, you are right. This should only be the case for max memory usage.